src/sync.js

```diff
@@ -184,6 +184,11 @@ class Sync {
       case side == null:
         this.pouch.setLocalSeq(change.seq, callback)
         break
+      case (sideName === 'remote' && doc.toBeTrashed && !inRemoteTrash(doc)):
+        // File or folder was just deleted locally
+        this.trashLaterWithParentOrByItself(doc, side)
+        done()
```

Comment:
Probably just a TODO for later: the change is marked as done right now, but if later the operation fails (being offline, or the cozy-stack is upgrading, etc.), it won't be retried